docs/audio-graph.md

```diff
@@ -0,0 +1,23 @@
+# Audio Graph
+
+This is responsible for processing the correct entities in order of expected flow, essentially a scene graph for audio. Audio sources (synths, samples) can be added as leaf nodes to the graph, then modulated and blended before reaching the main output.
+
+## Document Model
+
+[This mind map](https://whimsical.com/project-HNvThW62PrN9Gp33UfAuRf@2Ux7TurymNHcs4gAfi4M) includes a directional flow of the levels of hierarchy inside of a Project.
+
+### Project
+
+The highest level in the model. This includes filename, metadata, and a main output channel (ch0).
```

Comment:
This could also include tempo but we need to think about how we want to allow tempo to be modulated & how that should fit in to other forms of automation within a `DeviceTrack`